### PR TITLE
Adds a tcpdump option for tracing salt-api requests

### PR DESCRIPTION
**Pull request by brejoc**

### CODE REVIEW

**Comment by brejoc** [comment]
@moio Sounds good and thanks for the links. I hope the latest commit has everything needed!

**Comment by brejoc** [comment]
Is this a real error or what do we see here? :thinking: 

```
Executing section whitespace...
|    | [NORMAL] SpaceConsistencyBear:
|    | Line contains following spacing inconsistencies:
|    | - No newline at EOF.
|----|    | /home/travis/build/uyuni-project/sumaform/salt/suse_manager_server/tcpdump.sls
|    |++++| /home/travis/build/uyuni-project/sumaform/salt/suse_manager_server/tcpdump.sls
|  18|  18|       - file: tcpdump.unit
|  19|  19|     - enable: True
|  20|  20| 
|  21|    |-{% endif %}
|    |  21|+{% endif %}
[INFO][13:15:18] Applied 'ShowPatchAction' on 'salt/suse_manager_server/tcpdump.sls' from 'SpaceConsistencyBear'.
Executing section cli...
The command "coala --ci" exited with 5.
```

File: salt/suse_manager_server/tcpdump.sls [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - user: root
+    - group: root
+
+tcpdump.service:
+  service.running:
+    - require:
+      - file: tcpdump.unit
+    - enable: True
+
+{% endif %}
```
Comment (moio) [inline salt/suse_manager_server/tcpdump.sls]: Please add a newline at end of file and coala will stop complaining :wink:

File: modules/libvirt/host/main.tf [hunk 2]
```diff
@@ -98,6 +98,7 @@ gpg_keys: [${join(", ", formatlist("'%s'", var.gpg_keys))}]
 connect_to_base_network: ${var.connect_to_base_network}
 connect_to_additional_network: ${var.connect_to_additional_network}
 reset_ids: true
+saltapi_tcpdump: ${var.saltapi_tcpdump}
```
Comment (moio) [inline modules/libvirt/host/main.tf]: Please move this grain to libvirt/suse_manager/main.tf

**Comment by brejoc** [comment]
@moio I hope everything is in the right place now.

**Comment by moio** [comment]
It's good to go, just fix conflicts by rebasing on current master please

**Pull request merged**